test/controllers/sign.out.controller.spec.ts

```diff
@@ -85,8 +87,8 @@ describe("Sign Out controller", () => {
 
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(SIGN_OUT_PAGE_TITLE);
-      expect(resp.text).not.toContain(SIGN_OUT_HINT_TEXT);
-      expect(resp.text).not.toContain(SIGN_OUT_HELP_DETAILS_TEXT);
+      expect(resp.text).not.toContain(SIGN_OUT_SAVE_AND_RESUME_GUIDANCE_TEXT);
```

Comment:
Same here. I would revert the changes in these two lines but change line 93 (and also add a new one) to reference your new constants. Thanks.